upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -339,7 +339,11 @@ func NewEC2Filter(name string, values ...string) *ec2.Filter {
 // DeleteGroup deletes an aws autoscaling group
 func (c *awsCloudImplementation) DeleteGroup(g *cloudinstances.CloudInstanceGroup) error {
 	if c.spotinst != nil {
-		return spotinst.DeleteInstanceGroup(c.spotinst, g)
+		if featureflag.SpotinstHybrid.Enabled() {
+			if _, ok := g.Raw.(*autoscaling.Group); ok {
+				return deleteGroup(c, g)
```

Comment:
Is this right - it looks like we no longer call `spotinst.DeleteInstanceGroup` ?

Reply:
Good catch, I'm sure I did it, but... anyway, [fixed](https://github.com/kubernetes/kops/blob/163f878ee0653144d71684cfa564bf492071ba7a/upup/pkg/fi/cloudup/awsup/aws_cloud.go#L341-L349) as well.